setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.1.5
+current_version = 0.1.6
```

Comment:
I believe you do not have to bump version manually in both `setup.cfg` and `setup.py`, as one of GitHub Actions steps do it automatically.